app/src/main/java/org/literacyapp/synchronization/GroupDeleteHelper.java

```diff
@@ -63,12 +64,16 @@ public Object invoke(Object proxy, java.lang.reflect.Method method, Object[] arg
 											deletePersistentGroupMethod.invoke(mManager, mChannel, (Integer) WifiP2pGroup.class.getMethod("getNetworkId").invoke(group, null), new WifiP2pManager.ActionListener() {
 												@Override
 												public void onSuccess() {
-													Log.i(P.Tag, "Persistent Groups deleted");
+													if (isUI)
+														Log.i(P.Tag, "Persistent Group deleted");
+
 												}
 
 												@Override
 												public void onFailure(int i) {
-													Log.e(P.Tag, "Persistent Groups not deleted");
+													Log.e(P.Tag, "Persistent Group not deleted");
+													if (isUI)
+														Toast.makeText(ctx,"Persistent Group delete failed",Toast.LENGTH_LONG).show();
```

Comment:
Thanks for your review.
Toasts will not appear when used by the children since they will be blocked by the isUI param.
I need the toasts internally, for debugging.